Automattic/wp-calypso
# Checkout-composite: Add terms and conditions

#### Changes proposed in this Pull Request

This PR adds the terms and conditions copy to the review step. It conditionally shows terms bullets if there are domains in the cart. 

![image](https://user-images.githubusercontent.com/6981253/68359734-03d7a480-00eb-11ea-894f-28abdd311c2b.png)



#### Testing instructions
Look at screenshot or

* Get up and running: https://github.com/Automattic/wp-calypso/pull/37013
* Complete all the steps and see terms in the review step.
